### DISCUSSION TITLE
Second login

### DISCUSSION THREAD

**Discussion by ivodam2** [SauceLLC/sauce4zwift#16] [General]
May be question for dummies, but where to create second login. In Zwift or Sauce? I tried w/o second login and in riding with group everybody dissapeared.

**Comment by mayfield** [comment]
Hi @ivodam2 on the 2nd login window there is a link to the signup page.. 
![image](https://user-images.githubusercontent.com/139316/193878693-c9ff219f-de33-4a4f-9379-c9a232f9dbc4.png)

That will open a new window to https://www.zwift.com/create-account

Just fill out the first form, you don't need to go beyond that.
